src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -59,6 +60,9 @@ def __init__(self, job_id, initial_params, area, export_plots=True):
         self.area_throughput_stats = AreaThroughputStats()
 
         self.last_unmatched_loads = {}
+        if export_plots:
+            self.export_plots = export_plots
```

Comment:
You need to move this outside of the if block, otherwise `self.export_plots` will not be populated in the `update_stats` method